src/plugin.js

```diff
@@ -206,6 +206,9 @@ const contribAdsPlugin = function(options) {
 
   player.ads.adMacroReplacement = adMacroReplacement.bind(player);
 
+  // Expose snapshot tools for testing
+  player.ads._snapshot = snapshot;
```

Comment:
What I meant was: is it possible to import the snapshot module into the test file?